sql/core/src/test/scala/org/apache/spark/sql/scripting/SqlScriptingExecutionNodeSuite.scala

```diff
@@ -72,11 +72,39 @@ class SqlScriptingExecutionNodeSuite extends SparkFunSuite with SharedSparkSessi
     }
   }
 
+  case class TestRepeatCondition(
```

Comment:
nit: `TestWhileCondition` and `TestRepeatCondition` are exactly the same. Let's rename to `TestLoopCondition` or something similar to avoid duplicates.